sentry_sdk/profiler/transaction_profiler.py

```diff
@@ -60,15 +60,20 @@
     from typing import List
     from typing import Optional
     from typing import Set
-    from typing import Sequence
-    from typing import Tuple
     from typing_extensions import TypedDict
 
-    import sentry_sdk.tracing
+    from sentry_sdk.profiler.utils import (
```

Comment:
@sentrivana I moved this file into its own folder as suggested. One thing that I am wondering here is if this is considered a breaking change since users may be importing from it. This entire file was originally meant to be purely internal, and users should not be using it directly so I assume it's fine but just wanted to double check because I ran into this previously when I moved an utility some time ago.

Reply:
Yeah that's a good point. We're not explicitly marking this as internal, but at the same time, the public profiler API is basically what's exposed via `sentry_sdk.init` options. @antonpirker what do you think? (TL;DR: We're moving the contents of `profiler.py` to `profiler/transaction_profiler.py`.)

I'm thinking if it makes sense to expose more stuff from this file in `profiler/__init__.py` so that at least if someone is importing from `sentry_sdk.profiler` for whatever reason, the interface is more or less the same.

Reply:
I did a quick [github code search](https://github.com/search?type=code&auto_enroll=true&q=%22from+sentry_sdk.profiler%22) and only a hand full of people seem to import it (others are just forks of the SDK)

But to be on the safe side, we should do what @sentrivana said and expose the old API in `profiler/__init__.py`.